polkadot/xcm/xcm-builder/src/fungibles_adapter.rs

```diff
@@ -59,7 +59,10 @@ impl<
 		let dest = AccountIdConverter::convert_location(to)
 			.ok_or(MatchError::AccountIdConversionFailed)?;
 		Assets::transfer(asset_id, &source, &dest, amount, Preserve)
-			.map_err(|e| XcmError::FailedToTransactAsset(e.into()))?;
+			.map_err(|e| {
+				tracing::error!(target: "xcm::fungibles_adapter", error = ?e, ?amount, "Failed to transfer asset internally");
```

Comment:
```suggestion
				tracing::debug!(target: "xcm::fungibles_adapter", error = ?e, ?asset_id, ?source, ?dest, ?amount, "Failed internal transfer asset");
```

Do not use `error` log level unless critical error that is relevant to node operators - all of these non-critical runtime errors are ok to be `debug`, they're only relevant when debugging why some operation failed.

Also for debugging, provide as much information as possible about the context of the failure/error.